creator-node/src/services/sync/syncUtil.ts

```diff
@@ -37,6 +52,30 @@ type FetchExportOutput = {
       | 'abort_mismatched_export_wallet'
   }
 }
+type SyncStatus =
```

Comment:
is this some subset of the ISSUE_SYNC_REQUEST_DURATION_SECONDS_HISTOGRAM values? how'd you determine which to put here

Reply:
I used every result from `SECONDARY_SYNC_FROM_PRIMARY_DURATION_SECONDS_HISTOGRAM` because that's the result that's set in redis by the processing node to be read by the issuing node. then I added "waiting" as the default state

Reply:
what if those values change? we now have duplicate literals - should we instead pull from there